src/main/java/com/paneedah/weaponlib/grenade/GrenadeAttackAspect.java

```diff
@@ -215,7 +215,7 @@ public static void serverThrowGrenade(ModContext modContext, EntityLivingBase pl
         if(activationTimestamp == 0 && instance.getWeapon().getType() == Type.REGULAR) {
             // explode immediately
         	player.attackEntityFrom(DamageSource.causeExplosionDamage(player), 500f);
-            Explosion.createServerSideExplosion(modContext, player.world, null,
+            Explosion.createServerSideExplosion(modContext, player.world, null, null,
```

Comment:
Why null?

Reply:
This doesn't seem to affect the grenades, Because the killing caused by hand grenades can indeed obtain the thrower

Reply:
What is the arg name and what is it used for?

Reply:
What is arg name

Reply:
> What is arg name

https://www.w3schools.com/java/java_methods_param.asp

Reply:
I didn't see the parameter type above lol